requirements-test.txt

```diff
@@ -1,10 +1,10 @@
 # Additional requirements for the unit and coverage tests
 
-# Constraints on the requirements below
--c constraints.txt
+pycparser
+idna
 
 unittest2
-coveralls<=1.2.0
-mock<4.0.0  # Pinned because version 4 dropped support for Python 2.7
```

Comment:
Removing the constraint is likely to break the Travis Python 2 build (which the requirements files are mostly used for).

Reply:
(Same respone as before) I have added the separate flow for Travis now it should work as expected. Aimed to test codebase with py2 and haven't disturbed the previous libs with version info limitations.